setup.cfg

```diff
@@ -42,7 +42,7 @@ reqsfilenamepatterns =
 
 reqsignorevcs = True
 
-timeout = 60
```

Comment:
this needs to be 60, otherwise AppVeyor fails frequently -- there should be a .moban.yaml config variable for this.

Reply:
There's already a variable defined in [coala.yml](https://gitlab.com/coala/mobans/blob/master/coala.yaml#L31) which sets the timeout. DO I need to time here or still create a new variable in `.moban.yaml` ?

Reply:
Just set the variable in this projects .moban.yaml